### PR TITLE
Add google tag manager script

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What's the purpose of this pull request?
Add the google tag manager script tag if the tag manager id is set on the portal, like what's being done on the old platform.

#### How do I manually test this?
[Access this workspace](https://datalayer--storecomponents.myvtex.com/). You can see that after the request is done, the script tag is added to the header and a debug button is added on the lower left corner.

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [x] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/core/main.tsx [hunk 1]
```diff
@@ -109,6 +109,30 @@ function start() {
           state: getState(runtime),
         }))
     } else {
+      const {account: accountName} = window.__RUNTIME__
+
+      const { tagManagerId } = await fetch(
+        // `/api/configurationRepository/pvt/bucket/portal/key/TagManagerId_${accountName}?an=${accountName}`,
```
Comment (ClaudivanFilho) [inline react/core/main.tsx]: remove comment

File: manifest.json [hunk 2]
```diff
@@ -19,5 +19,20 @@
   "settingsSchema": {},
   "scripts": {
     "postreleasy": "vtex publish --verbose"
-  }
+  },
+  "policies": [
```
Comment (ClaudivanFilho) [inline manifest.json]: remove these policies, are not necessary

File: react/core/main.tsx [hunk 3]
```diff
@@ -109,6 +109,30 @@ function start() {
           state: getState(runtime),
         }))
     } else {
+      const {account: accountName} = window.__RUNTIME__
+
+      const { tagManagerId } = await fetch(
```
Comment (ClaudivanFilho) [inline react/core/main.tsx]: Better to use axios.
When u use it, `{ headers: { Accept: 'application/json' }` and `.then(res => res.json())` are not necessary

Comment (andreldsa) [reply react/core/main.tsx]: use axios

File: react/core/main.tsx [hunk 4]
```diff
@@ -109,6 +110,29 @@ function start() {
...
+
+          if (firstScriptTag.parentNode) {
+            firstScriptTag.parentNode.insertBefore(script, firstScriptTag)
+          } else {
+            document.head.insertBefore(script, document.head.firstChild)
+          }
+        }
+      }
+
       console.log('Welcome to Render! Want to look under the hood? http://lab.vtex.com/careers/')
```
Comment (brunojdo) [inline react/core/main.tsx]: change the URL to https://careers.vtex.com/

**Comment by lucasecdb** [comment]
Closing in favor of vtex-apps/store#42

**Pull request closed without merging**